azure-quantum/azure/quantum/target/ionq.py

```diff
@@ -103,13 +106,28 @@ def submit(
             )
         if input_params is None:
             input_params = {}
+
+        num_shots = kwargs.pop("num_shots", None)
+
         if num_shots is not None:
+            warn(
+                "The 'num_shots' will be deprecated. Please use 'shots' instead.",
```

Comment:
super-nit: just to be more specific in messaging and follow the pattern defined on the line 105
```suggestion
                "The 'num_shots' parameter will be deprecated. Please, use 'shots' parameter instead.",
```